test/unit/sync/branch.test.ts

```diff
@@ -1,29 +1,26 @@
 /* eslint-disable @typescript-eslint/no-var-requires,@typescript-eslint/no-explicit-any */
 import issueKeyParser from "jira-issue-key-parser";
-import {branchesNoLastCursor} from "../../fixtures/api/graphql/branch-queries";
-import {mocked} from "ts-jest/utils";
-import {Installation, RepoSyncState, Subscription} from "../../../src/models";
-import {Application} from "probot";
-import {createWebhookApp} from "../../utils/probot";
-import {processInstallation} from "../../../src/sync/installation";
+import { branchesNoLastCursor } from "../../fixtures/api/graphql/branch-queries";
+import { mocked } from "ts-jest/utils";
+import { Installation, RepoSyncState, Subscription } from "../../../src/models";
+import { Application } from "probot";
+import { createWebhookApp } from "../../utils/probot";
+import { processInstallation } from "../../../src/sync/installation";
 import nock from "nock";
-import {getLogger} from "../../../src/config/logger";
-import {Hub} from "@sentry/types/dist/hub";
-import {BackfillMessagePayload} from "../../../src/sqs/backfill";
-import sqsQueues from "../../../src/sqs/queues";
-import {when} from "jest-when";
-import {booleanFlag, BooleanFlags} from "../../../src/config/feature-flags";
-
-jest.mock("../../../src/sqs/queues", () => {
-	return {
-		backfill: {sendMessage: jest.fn()}
-	}
-});
-
+import { getLogger } from "../../../src/config/logger";
+import { Hub } from "@sentry/types/dist/hub";
+import { BackfillMessagePayload } from "../../../src/sqs/backfill";
+import { sqsQueues } from "../../../src/sqs/queues";
+import { when } from "jest-when";
+import { booleanFlag, BooleanFlags } from "../../../src/config/feature-flags";
+
+jest.mock("../../../src/sqs/queues");
 jest.mock("../../../src/config/feature-flags");
 
-describe("sync/branches", () => {
+
+describe.each([true, false])("sync/branches - New GH Client feature flag is '%s'", () => {
 	const installationId = 1234;
+	const useNewGithubClient = true;
```

Comment:
It will be constantly true now. Should we set it to the describe parameter value?

```suggestion
	describe.each([true, false])("sync/branches - New GH Client feature flag is '%s'", (useNewGithubClient) => {
	const installationId = 1234;
```

Reply:
- [ ] fix feature flag mocking

Reply:
whoops, forgot to change that back after debugging.